src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -224,26 +224,37 @@ protected void setRemoteAuth(String remoteAuth) {
         this.remoteAuth = remoteAuth;
     }
 
+    @JsonProperty
+    @Parameter(names = {"-nsa", "--neighboring-socket-address"}, description = NetworkConfig.Descriptions.NEIGHBORING_SOCKET_ADDRESS)
+    public void setNeighboringSocketAddress(String neighboringSocketAddress) {
+        this.neighboringSocketAddress = neighboringSocketAddress;
+    }
+
     @Override
-    public int getUdpReceiverPort() {
-        return udpReceiverPort;
+    public String getNeighboringSocketAddress() {
+        return neighboringSocketAddress;
     }
 
     @JsonProperty
-    @Parameter(names = {"-u", "--udp-receiver-port"}, description = NetworkConfig.Descriptions.UDP_RECEIVER_PORT)
-    public void setUdpReceiverPort(int udpReceiverPort) {
-        this.udpReceiverPort = udpReceiverPort;
+    @Parameter(names = {"-nsp", "--neighboring-socket-port"}, description = NetworkConfig.Descriptions.NEIGHBORING_SOCKET_PORT)
```

Comment:
This doesn't follow [conventions concerning command line arguments](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) since you're using multiple characters to represent an option but only a single hyphen `-`. Maybe use `-s` (since `-n` and `-p` are already used)?

Reply:
Or capital letter